turingschool-examples/monster_shop_2005
# Login and Logout

Testing
  * Add logout functionality
  * Add test to check the inability to relog in as a user 
 MCV 
  * Add destroy method to Sessions Controller
  * Add conditionals to allow for redirects for specific users
  * Change a route to logout in applications.html
ROUTE
  * DELETE route for sessions#destroy
ADDITIONAL
  * Add nyancatt for testing
